libpdbg/sbefifo.c

```diff
@@ -197,6 +197,37 @@ static uint32_t sbefifo_op_ffdc_get(struct chipop *chipop, const uint8_t **ffdc,
 
 }
 
+static uint32_t sbefifo_op_ody_ffdc_get(struct pdbg_target *ocmb, const uint8_t **ffdc, uint32_t *ffdc_len)
+{
+	struct pdbg_target *fsi = get_ody_fsi_target(ocmb);
+	struct sbefifo *sbefifo = target_to_sbefifo(ocmb);
+	
+	struct sbefifo_context *sctx = sbefifo->get_sbefifo_context(sbefifo);
+	uint32_t status, value = 0;
+	int rc;
+
+	status = sbefifo_ffdc_get(sctx, ffdc, ffdc_len);
```

Comment:
should it be sbe_ffdc_ody_get?

Reply:
for low level communication we still use existing methods